privacyidea/api/lib/postpolicy.py

```diff
@@ -503,7 +504,7 @@ def get_webui_settings(request, response):
             realm=realm,
             client=client
         )
-        timeout_action = policy_object.get_policies(
+        timeout_action_pol = policy_object.get_policies(
```

Comment:
A default value will not work. If the policy is set to 5 seconds, the problem will arise in javascript. 

For now fixed it this way:
https://github.com/privacyidea/privacyidea/blob/master/privacyidea/static/components/login/controllers/loginControllers.js#L283